cli/tests/unit/fetch_test.ts

```diff
@@ -1149,3 +1149,20 @@ unitTest({}, function fetchWritableRespProps(): void {
   assertEquals(original.status, new_.status);
   assertEquals(new_.headers.get("x-deno"), "foo");
 });
+
+unitTest(
+  { perms: { net: true } },
+  async function fetchFilterOutCustomHostHeader(): Promise<
+    void
+  > {
+    const client = Deno.createHttpClient({});
```

Comment:
You can remove this client creation and closing a few lines down. It is unused and not needed.

Reply:
Done!